frontend/packages/dev-console/src/components/import/image-search/ImageSearch.scss

```diff
@@ -0,0 +1,8 @@
+.odc-image-search {
+  &_help-text-success {
+    color: var(--pf-global--success-color--200);
+  }
+  &_help-text-invalid {
+    color: var(--pf-global--danger-color--200);
+  }
```

Comment:
These should not be necessary. Patternfly correctly styles the helper text.

Reply:
removed stylesheet.